mysql-test/suite/tokudb/t/locking-read-repeatable-read-1.test

```diff
@@ -0,0 +1,27 @@
+source include/have_tokudb.inc;
+
+SET DEFAULT_STORAGE_ENGINE='tokudb';
+
+--disable_warnings
+drop table if exists t;
+--enable_warnings
+create table t (a int primary key, b int);
```

Comment:
Adding to Laurynas 'obsolete idiom', we should not use DEFAULT_STORAGE_ENGINE, tables should be explicitly created with ENGINE='<engine>' for clarity.